lib/Rx/React/Promise.php

```diff
@@ -76,7 +76,10 @@ function ($value) use ($subject) {
                         $subject->onNext($value);
                         $subject->onCompleted();
                     },
-                    [$subject, "onError"]
+                    function ($error) use ($subject) {
+                        $error = $error instanceof \Exception ? $error : new RejectedPromiseException($error);
```

Comment:
Should we be chaining exceptions, so we're always calling onError with `RejectedPromiseException`?